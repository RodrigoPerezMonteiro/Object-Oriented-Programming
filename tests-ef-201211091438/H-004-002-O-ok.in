testH4.dat
1
0
testH4b.dat
