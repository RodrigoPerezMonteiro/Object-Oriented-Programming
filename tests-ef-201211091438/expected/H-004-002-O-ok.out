Ficheiro a abrir: Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: VEG|alface
NOVEG|linguado grelhado
VEG|salada mista
Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Ficheiro a guardar: 